### DISCUSSION TITLE
Kafka Integration with Vault for secret

### DISCUSSION THREAD

**Discussion by stuartm21** [strimzi/strimzi-kafka-operator#6589] [Q&A] [answered]
Hi Team,
We are looking to integrate the kafka secret with Vault. While searching we found the two following docs, but we noticed that both are 2 and 4 years old. So any latest docs or solutions for integrating our kafka-users secrets with vault setup?

1. https://github.com/strimzi/strimzi-kafka-operator/issues/3719
2. https://strimzi.io/blog/2018/11/16/using-vault-with-strimzi/

**Comment by scholzj** [comment, marked as answer]
There is currently no special support for Vault as part of Strimzi.

**Reply by stuartm21** [reply]
@scholzj  That means the provided URL's will not support Strizmzi kafka right? We are using cluster operator 0.28 and 3.1.0 Kafka.

**Reply by scholzj** [reply]
Sorry, I have no idea what kind of _provided URL's_ mean here.

**Reply by stuartm21** [reply]
@scholzj  These two:

https://github.com/strimzi/strimzi-kafka-operator/issues/3719
https://strimzi.io/blog/2018/11/16/using-vault-with-strimzi/

**Reply by scholzj** [reply]
Well, if you read the blog post, it does not mention any direct integration with Strimzi, any special APis or anything like that -> the integration happens through Kubernetes Secrets. In such way you can integrate with many things. So while the blog post is old and many details might have changed the idea might still work.
